Deploy/stacks/db/fileserver/fileserver/src/main/java/com/cmclinnovations/fileserver/FileServer.java

```diff
@@ -0,0 +1,198 @@
+package com.cmclinnovations.fileserver;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URLDecoder;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ *
+ * @author Owen Parry {@literal <oparry@cmclinnovations.com>}
+ */
+@WebServlet(name = "FileServer", urlPatterns = {FileServer.DELETE_URL_PATTERN, FileServer.DOWNLOAD_URL_PATTERN, FileServer.UPLOAD_URL_PATTERN, FileServer.UPLOAD_URL_PATTERN + "/"})
+@MultipartConfig(
+fileSizeThreshold = FileServer.ONE_MB_IN_B,
+location = "/app/fs_root/",

```

Comment:
Make it possible to set the file size settings via environment variables.

Reply:
These settings have been moved to the web.xml file and are populated during the Maven build stage.